mail_debrand/models/mail_template.py

```diff
@@ -21,7 +21,11 @@ def generate_email(self, res_ids, fields=None):
 
     @api.model
     def _debrand_body(self, body):
-        using_word = _('using')
+        """Depending of the context the term could come translated or in
+           the original term. We should consider both. Also notice that if
+           the translated term in this module differs from the one in the
+           template it won't be catched."""
+        using_word = "[using|{}]".format(_('using'))
```

Comment:
```suggestion
        using_word = "(using|{})".format(_('using'))
```

I'm wondering if it would be possible to obtain the translation by manually searching the `ir.translation` table... :thinking:

Reply:
I don't understand your suggestion

Reply:
The code suggestion or the suggestion to search it from `ir.translation`?

Reply:
The code one

Reply:
Oh ok. Before, it matched "gnisu": https://regex101.com/r/sUHwg4/1
After, it doesn't: https://regex101.com/r/1yKYFo/1/
That's because brackets are character groups, while parenthesis are sub-regexs.

Reply:
About exploring the ir.translation option, I think that can be interesting for avoiding to re-translate the word on each language and avoid inconsistencies.

Reply:
> brackets are character groups, while parenthesis are sub-regexs.

Sure :)

Reply:
> About exploring the ir.translation option, I think that can be interesting for avoiding to re-translate the word on each language and avoid inconsistencies.

But in email templates the terms aren't separeted and they are in views. The translations is a copy of the whole original code.

Reply:
OK, I see...

https://github.com/odoo/odoo/blob/8c1bb22ec0222dca652e0649454b986c06cb1368/addons/mail/i18n/mail.pot#L102

Maybe in v12 will it be possible?

Reply:
> Maybe in v12 will it be possible?
Well, in v12 is an updatable qweb view, so I guess the containing tag can be just replaced or hidden via xpath.